### ISSUE TITLE
recovered

### ISSUE THREAD

**Issue by rwkincaid** [CSSEGISandData/COVID-19#2437]
I am specifically looking for data on recoveries from the virus, broken down by date and state in the USA. Was going along fine until about 3/17, when suddenly all "recovered" in the USA went to 0. I'm looking forward in data to see if it comes back, but up to 4/17 and haven't found anything but 0 in between for recovered in USA

**Comment by akshayprograms** [comment]
Same issue I am facing data is not matching with actual records. 
Also recovered and active data columns are not available for US from Jan22nd till 21st March.
Even if we can get recovered data from Jan22nd till 21st March that will be fine.

**Comment by JamesZFS** [comment]
Yeah, same situation happens to me. It seems that the recovered data coalesce into a "Recovered" row which sum ups all the recovered data across US.
